### PR TITLE
Remove old wallet package from dev-dependencies

### PR DESCRIPTION
**Pull request by popenta**
Removed old wallet package (v3.0.0) and replaced `sdk-wallet-next(v4.0.0)` to `sdk-wallet(v4.2.0)`.

### CODE REVIEW

File: src/testutils/wallets.ts [hunk 1]
```diff
@@ -73,7 +72,7 @@ export class TestWallet {
     readonly secretKeyHex: string;
     readonly secretKey: Buffer;
     readonly signer: UserSigner;
-    readonly signerNext: UserSignerNext;
+    readonly signerNext: UserSigner;
```
Comment (andreibancioiu) [inline src/testutils/wallets.ts:75]: I think can be dropped now.

File: src/testutils/wallets.ts [hunk 2]
```diff
@@ -83,7 +82,7 @@ export class TestWallet {
         this.secretKeyHex = secretKeyHex;
         this.secretKey = Buffer.from(secretKeyHex, "hex");
         this.signer = new UserSigner(UserSecretKey.fromString(secretKeyHex));
-        this.signerNext = new UserSignerNext(UserSecretKey.fromString(secretKeyHex));
+        this.signerNext = new UserSigner(UserSecretKey.fromString(secretKeyHex));
```
Comment (andreibancioiu) [inline src/testutils/wallets.ts:85]: Now, a duplicate of the above.

File: src/testutils/wallets.ts (hunk 1 above)
Comment (popenta) [reply src/testutils/wallets.ts:75]: Removed `signerNext`

File: src/testutils/wallets.ts (hunk 2 above)
Comment (popenta) [reply src/testutils/wallets.ts:85]: removed

**Review by andreibancioiu — APPROVED** [review]

**Review by MiroMargineanu — APPROVED** [review]

**Pull request merged**